client/components/Channel.vue

```diff
@@ -9,6 +9,11 @@
 			:class="{ highlight: channel.highlight }"
 			class="badge">{{ channel.unread | roundBadgeNumber }}</span>
 		<template v-if="channel.type === 'channel'">
+			<span
```

Comment:
Hide this with v-if

Reply:
I did a thing, can you check if that's what you meant? 🙏

Reply:
Yep. We'll want to do this with other icons too (the ones you moved styles for).